style.css

```diff
@@ -0,0 +1,3 @@
+body {
```

Comment:
[ ] the styling is for your `h1` text, kindly fix that since it's part of the requirements 🤓.

![header](https://user-images.githubusercontent.com/62156607/197747741-a73cf737-a622-44d7-97a3-459db6187844.PNG)

Reply:
done adding style to h1 tag